torchmetrics/collections.py

```diff
@@ -45,6 +44,14 @@ class name as key for the output dict.
 
         postfix: a string to append after the keys of the output dict
 
+        compute_groups:
+            By defualt the MetricCollection will try to reduce the computations needed for the metrics in the collection
```

Comment:
```suggestion
            By default the MetricCollection will try to reduce the computations needed for the metrics in the collection
```

again :D

Reply:
I see how you enjoy it in your office - just find anything :D

Reply:
I just looked through it again since it popped up in my notifications, but yes, I quite enjoyed it when I saw this :P